specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -502,7 +502,430 @@
+          },
+          {
+            "value": 13,
+            "name": "Mostly Cloudy with Showers",
+            "description": "Mostly Cloudy with Showers"
+          },
+          {
+            "value": 14,
+            "name": "Partly Sunny with Showers",
+            "description": "Partly Sunny with Showers"
+          },
+          {
+            "value": 15,
+            "name": "Thunderstorms",
+            "description": "Thunderstorms"
+          },
+          {
+            "value": 16,
+            "name": "Mostly Cloudy with Thunderstorms",
+            "description": "Mostly Cloudy with Thunderstorms"
+          },
+          {
+            "value": 17,
+            "name": "Partly Sunny with Thunderstorms",
+            "description": "Partly Sunny with Thunderstorms"
+          },
+          {
+            "value": 18,
+            "name": "Rain",
+            "description": "Rain"
+          },
+          {
+            "value": 19,
+            "name": "Flurries",
+            "description": "Flurries"
+          },
+          {
+            "value": 20,
+            "name": "Mostly Cloudy with Flurries",
+            "description": "Mostly Cloudy with Flurries"
+          },
+          {
+            "value": 21,
+            "name": "Partly Sunny with Flurries",
+            "description": "Partly Sunny with Flurries"
+          },
+          {
+            "value": 22,
+            "name": "Snow",
+            "description": "Snow"
+          },
+          {
+            "value": 23,
+            "name": "Mostly Cloudy with Snow",
+            "description": "Mostly Cloudy with Snow"
+          },
+          {
+            "value": 24,
+            "name": "Ice",
+            "description": "Ice"
+          },
+          {
+            "value": 25,
+            "name": "Sleet",
+            "description": "Sleet"
+          },
+          {
+            "value": 26,
+            "name": "Freezing Rain",
+            "description": "Freezing Rain"
+          },
+          {
+            "value": 29,
+            "name": "Rain and Snow",
+            "description": "Rain and Snow"
+          },
+          {
+            "value": 30,
+            "name": "Hot",
+            "description": "Hot"
+          },
+          {
+            "value": 31,
+            "name": "Cold",
+            "description": "Cold"
+          },
+          {
+            "value": 32,
+            "name": "Windy",
+            "description": "Windy"
+          },
+          {
+            "value": 33,
+            "name": "Clear",
+            "description": "Clear"
+          },
+          {
+            "value": 34,
+            "name": "Mostly Clear",
+            "description": "Mostly Clear"
+          },
+          {
+            "value": 35,
+            "name": "Partly Cloudy",
+            "description": "Partly Cloudy"
+          },
+          {
+            "value": 36,
+            "name": "Intermittent Clouds",
```

Comment:
Duplicated with `"value": 4`.
Other duplicates: (1) 6 and 38, (2) 16 and 42 (3) 20 and 43 (4) 23 and 44 (5) 34 and 40.

TypeScript SDK build will fail because of the duplicated values.

Reply:
updated, thanks for checking the JS for me.